CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 🏉 [4.19.2](https://github.com/ionic-team/stencil/compare/v4.19.1...v4.19.2) (2024-07-02)
+
+
+### Bug Fixes
+
+* **hydrate:** partially revert [#5838](https://github.com/ionic-team/stencil/issues/5838) ([#5876](https://github.com/ionic-team/stencil/issues/5876)) ([dfbc340](https://github.com/ionic-team/stencil/commit/dfbc34007a818eef418e2f312a9dd7a0fef81af6))
```

Comment:
We wanna link any fixes for this one?

Reply:
I don't have a direct reported issue from within the Stencil project, this might have been reported in Ionic but I don't see that we reference those.